packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -58,7 +59,9 @@ export default class PciStoragesContainersService {
           endpoints,
           (result, endpoint) => ({
             ...result,
-            [endpoint.region.toLowerCase()]: endpoint.url,
+            [(
+              endpoint.region || OPENIO_DEFAULT_REGION
```

Comment:
```suggestion
              endpoint.region
```
Already handled by the model, no ?